evalml/tests/automl_tests/test_automl.py

```diff
@@ -300,11 +302,11 @@ def test_pipeline_limits(
         automl.search()
     out = caplog.text
     if verbose:
-        assert "Using default limit of max_batches=4." in out
-        assert "Searching up to 4 batches for a total of" in out
+        assert "Using default limit of max_batches=3." in out
```

Comment:
Had to change because ensembling is turned off by default in `AutoMLSearch`.

Reply:
Let's remember to update our perf test code! I think we still want the ability to test the ensembler pipeline in batch 4 when we kick off a job?